{
  target: {
    requestScopeIdentifier: {
      ScopeId: Guid_1
    },
    Instances: [
      {
        ScopeId: Guid_2,
        Name: MockAsyncEventHandler1,
        HandlerStatus: Done,
        IsDisposed: true
      },
      {
        ScopeId: Guid_2,
        Name: MockAsyncEventHandler2,
        HandlerStatus: Faulted,
        IsDisposed: true
      },
      {
        ScopeId: Guid_2,
        Name: MockAsyncEventHandler3,
        HandlerStatus: Done,
        IsDisposed: true
      }
    ]
  },
  logs: [
    {
      Error: An exception was thrown while executing Reprise.UnitTests.Features.MockAsyncEventHandler.,
      State: [
        {
          EventHandlerType: Reprise.UnitTests.Features.MockAsyncEventHandler
        },
        {
          {OriginalFormat}: An exception was thrown while executing {EventHandlerType}.
        }
      ],
      Exception: {
        Type: Exception,
        Message: Test message
      }
    }
  ]
}